src/pages/About.js

```diff
@@ -1,7 +1,50 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import Card from '@mui/material/Card';
+import Box from '@mui/material/Box';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import { CardActionArea } from '@mui/material';
+import { getRequest } from '../services/requestsHandlerService';
 
 function About() {
-  return <h1>About us...</h1>;
+  const [members, setMembers] = useState([]);
+  useEffect(()=> {
+    getRequest('/members')
+      .then(result => {
+        setMembers(result);
+      })
+      .catch(err => {
+        console.log(err)
+      })
+  }, [])
+  return (
+    <Box 
+    sx={window.innerWidth >= '768'? {
+      display:'flex',
+      margin: 'auto'
+    }: {
+      margin: 'auto'
+    }}>
+      {members.map((member, i) => 
+        <Card sx={{ maxWidth: 345, margin: '10px 20px' }} key={i}>
+          <CardActionArea>
+            <CardMedia
+              component="img"
+              height="140"
+              image={member.image}
+              alt="green iguana"
```

Comment:
No se si a los miembros le va a gustar que le digamos iguanas verdes..jajaja

Reply:
jaja eso quedo del copy and paste del codigo. Lo saco.